framework/packages/abstract-interface/src/account/manager.rs

```diff
@@ -61,6 +61,19 @@ impl<Chain: CwEnv> Manager<Chain> {
         Ok(())
     }
 
+    pub fn install_modules<TInitMsg: Serialize>(
+        &self,
+        modules: Vec<(ModuleInfo, Option<&TInitMsg>)>,
+        funds: Option<&[Coin]>,
+    ) -> Result<Chain::Response, crate::AbstractInterfaceError> {
+        let modules = modules
+            .into_iter()
+            .map(|(module, init_msg)| (module, init_msg.map(|msg| to_binary(msg).unwrap())))
+            .collect();
+        self.execute(&ExecuteMsg::InstallModules { modules }, funds)
```

Comment:
Maybe if Options = None it would query the factory for the cost and use that by default?

Reply:
maybe not lol

Reply:
We can make something like
```rust
enum InstallModulesFunds {
  Manual(Vec<Coin>),
  Auto,
}
```
and default (not sure which tbh)

Reply:
maybe a separate method is easier